WordPress/Classes/ViewRelated/Reader/Select Interests/Mock Data/InterestsDataSource.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+
+class ReaderInterest {
+    var isSelected: Bool = false
+
+    let title: String
+    let slug: String
+
+    init(title: String, slug: String) {
+        self.title = title
+        self.slug = slug
+    }
+}
+
+class InterestsDataSource {
```

Comment:
I'm not sure if you pretend to use `InterestsDataSource` in the final implementation, but we can add some unit tests here to make sure that given a JSON this outputs the correct list of interests.